micropython/senml/README.md

```diff
@@ -0,0 +1,15 @@
+# Introduction
+
+The KPN SenML library helps you create and parse [senml documents](https://tools.ietf.org/html/draft-ietf-core-senml-13)
+in both json and cbor format.
+
+# key features
+
+- Object oriented design.
+- built in support for [senml's unit registry](https://tools.ietf.org/html/draft-ietf-core-senml-12#section-12.1)
+- extensible for new data types
+- makes use of (but doesn't restrict to) KPN's predefined list of record names.
```

Comment:
I don't think we should include KPN's predefined list of record names.  As per the discussion in the main thread of this PR, this takes up space and is very specificy to KPN.  Users can define and use their own record names.